packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -299,12 +299,17 @@ export const SidebarScrollWrapper = styled('div')(({ theme }) => ({
   // Display at least one item in the container
   // Question: Can this be a config/theme variable - if so, which? :/
   minHeight: '48px',
-  '&::-webkit-scrollbar': {
+  overflowY: 'hidden',
+
+  '&:hover': {
```

Comment:
only question here i guess is how this affects touch devices; would one want it to always be visible on those 🤔 

this stuff is hard :P in any case this is a nice improvement and we could always address that separately

Reply:
That's a good shout, I seem to remember that they might always be visible on touch devices by default, but I could be wrong and we should check this to confirm in a a trusty iPhone sim :)

Reply:
ahhhh - good catch. This always happens 🙆‍♂️ Should work now.